services/app/src/components/views/quest/cardtemplates/decision/Actions.test.tsx

```diff
@@ -1,39 +1,212 @@
+import {
+  extractDecision,
+  initDecision,
+  computeSuccesses,
+  computeOutcome,
+  generateLeveledChecks,
+  skillTimeMillis,
+  handleDecisionRoll,
+  toDecisionCard,
+} from './Actions';
+import {defaultContext} from '../Template';
+import {ParserNode} from '../TemplateTypes';
+import {EMPTY_DECISION_STATE} from './Types';
+import {Action, newMockStore} from 'app/Testing';
+import {initialMultiplayer} from 'app/reducers/Multiplayer';
+import {Outcome} from 'shared/schema/templates/Decision';
+
+const cheerio = require('cheerio');
+const seedrandom = require('seedrandom');
+
+const TEST_SETTINGS = {
```

Comment:
I've noticed that we seem to duplicate TEST_SETTINGS in a lot of places (which makes it less robust against adding / changing how settings work). Thoughts re: exporting this from the settings reducer?

Reply:
I'm not a fan of including test-only code in the same file as business logic... makes it more likely to be accidentally used in the wrong way. Added a common `TestData` file instead for the reducers directory, which is what we do in the api code when we need to test a bunch of different kinds of models.